Resources/Prototypes/_Goobstation/Balloons/balloons.yml

```
@@ -0,0 +1,376 @@
+  - type: Fixtures # give it locker collision
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.25,-0.48,0.25,0.48"
+        density: 75
+        mask:
+          - TabletopMachineMask
+        layer:
+          - TabletopMachineLayer
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)